### PR TITLE
Bugfix/credit card login input bug

### PR DESCRIPTION
**Pull request by petrKavulok**
#### In this PR we tackle
- issue, where Google Chrome assumes that login input is a credit card field

##### before
Screenshot 2022-09-20 at 14 41 27

##### after 
https://user-images.githubusercontent.com/79644454/193593491-e9a118b2-ffda-457e-b90b-f226f6994f2d.mov

### CODE REVIEW

File: src/modules/auth/passwd/ForgetPwdScreen.js [hunk 1]
```diff
@@ -35,9 +35,10 @@ function ForgetPwdCard(props) {
 	const [ completed, setCompleted ] = useState(false);
 	let invalidCode = getParams("invalid_code");
 
-	const identRegister = register('ident');
+	const usernameRegister = register('username');
 
-	const onSubmit = async (values) => {
+	const onSubmit = async (data) => {
+		let values = { ident: data.username };
```
Comment (Pe5h4) [inline src/modules/auth/passwd/ForgetPwdScreen.js]: @petrKavulok Why did you create a new `values` variable? Wouldnt be better just to setup the `values["ident"] = values.username`? And after that you just reset the `username` with `reset("username") or `unregister("username")` - dont remember which one removes it from the values.

File: src/modules/auth/passwd/ForgetPwdScreen.js [hunk 2]
```diff
@@ -35,9 +35,10 @@ function ForgetPwdCard(props) {
 	const [ completed, setCompleted ] = useState(false);
 	let invalidCode = getParams("invalid_code");
 
-	const identRegister = register('ident');
+	const usernameRegister = register('username');
 
-	const onSubmit = async (values) => {
+	const onSubmit = async (data) => {
```
Comment (Pe5h4) [inline src/modules/auth/passwd/ForgetPwdScreen.js]: @petrKavulok Pls return it back to original state ` onSubmit = async (values) => ...`

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Update submodule as well and test it with it pls.

File: src/modules/auth/passwd/ForgetPwdScreen.js (hunk 1 above)
Comment (Pe5h4) [reply src/modules/auth/passwd/ForgetPwdScreen.js]: for more, pls follow react hook form docu

File: src/modules/auth/containers/LoginCard.js [hunk 3]
```diff
@@ -268,10 +268,10 @@ function LoginCard(props) {
 		setSharedSecretKey(null);
 		setLoginButtonHidden(false);
 		reset({
-			ident: ""
+			username: ""
 		});
 
-		document.getElementById("ident").focus();
+		document.getElementById("username").focus();
```
Comment (Pe5h4) [inline src/modules/auth/containers/LoginCard.js:274]: @petrKavulok Pls check if `focus` works properly (it should focus to username input field automatically)

File: src/modules/auth/passwd/ForgetPwdScreen.js [hunk 4]
```diff
@@ -35,9 +35,11 @@ function ForgetPwdCard(props) {
 	const [ completed, setCompleted ] = useState(false);
 	let invalidCode = getParams("invalid_code");
 
-	const identRegister = register('ident');
+	const usernameRegister = register('username');
 
 	const onSubmit = async (values) => {
+		values.ident = values.username;
```
Comment (Pe5h4) [inline src/modules/auth/passwd/ForgetPwdScreen.js]: @petrKavulok I would stick to the value assignment as you have on line `47` to have it unified within the method

```
values["ident"] = values["username"];
delete values["username"];
```

Btw. also write some comment pls why we are doing this "substitution"

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Small comments, otherwise looks good!

File: src/modules/auth/containers/LoginCard.js (hunk 3 above)
Comment (petrKavulok) [reply src/modules/auth/containers/LoginCard.js:274]: it does :-)

File: src/modules/auth/passwd/ForgetPwdScreen.js [hunk 5]
```diff
@@ -35,9 +35,14 @@ function ForgetPwdCard(props) {
 	const [ completed, setCompleted ] = useState(false);
 	let invalidCode = getParams("invalid_code");
 
-	const identRegister = register('ident');
+	const usernameRegister = register('username');
 
 	const onSubmit = async (values) => {
+		// 'username' to 'ident' conversion is necessary here as the put request expects identity information key labeled
```
Comment (Pe5h4) [inline src/modules/auth/passwd/ForgetPwdScreen.js]: @petrKavulok Please wrap a multiline comment into this

```
/*
  my multiline comment
  second line
  third line
*/
```

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@petrKavulok Small issue, otherwise cool

**Review by Pe5h4 — APPROVED** [review]
@petrKavulok Looks good to me

**Review by ateska — APPROVED** [review]

**Pull request merged**